test/count.js

```diff
@@ -0,0 +1,16 @@
+/* global describe, it */
+import { assert } from 'chai'
+import Database from '../src/CRUD.js'
+
+let db = new Database()
+
+describe('Database `count()` function', () => {
+  it('should return the number of existing entries', () => {
+    let count = db.count()
+    db.insert({ foo: 'bar' })
+
+    assert(count === 0)
```

Comment:
If just using `assert` I don't think you need `chai`.

You could do `expect(count).to.equal(0)`, `expect(deletion).to.be.true`, `expect(db.delete(42)).to.be.undefined`, `expect(entry).to.be.a('number')`.

But I believe `assert` is enough, so maybe just use the native one?